presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -60,45 +68,41 @@ protected QueryRunner createQueryRunner()
     @Test
     public void testCteExecutionWhereOneCteRemovedBySimplifyEmptyInputRule()
     {
-        String sql = "WITH t as(select orderkey, count(*) as count from (select orderkey from orders where false) group by orderkey)," +
+        @Language("SQL") String sql = "WITH t as(select orderkey, count(*) as count from (select orderkey from orders where false) group by orderkey)," +
```

Comment:
I know we use this Language annotation in various places, but honestly I'm not a big fan of it. If I had my druthers, I'd drop it.

Reply:
I've removed the Language annotation as it wasn't originally used in this class. If someone objects my suggestion would be to open a new issue so we can decide then.